### PR TITLE
Use indexblob to determine versionbundle version

### PR DESCRIPTION
**Pull request by MarcelMue**
This uses the versionbundle version from the indexblob in our e2e tests.

### CODE REVIEW

File: integration/env/common.go [hunk 1]
```diff
@@ -31,6 +44,34 @@ func init() {
...
+	}
+
+	var err error
+	params := &framework.VBVParams{
+		Component: "chart-operator",
+		Provider:  "aws",
+		Token:     token,
+		VType:     TestedVersion(),
+	}
+	versionBundleVersion, err = framework.GetVersionBundleVersion(params)
```
Comment (MarcelMue) [inline integration/env/common.go]: This is aligned to `aws-operator` right now. I want to move this stuff out of env in the future, possibly into setup?

File: integration/test/basic/basic_test.go [hunk 2]
```diff
@@ -33,12 +34,11 @@ func TestChartLifecycle(t *testing.T) {
...
-		Name:      "tb-chart",
-		Namespace: "giantswarm",
-		Release:   "tb-release",
-		//TODO: fix this static VersionBundleVersion
-		VersionBundleVersion: "0.2.0",
+		Channel:              "5-5-beta",
+		Name:                 "tb-chart",
+		Namespace:            "giantswarm",
+		Release:              "tb-release",
+		VersionBundleVersion: env.VersionBundleVersion(),
```
Comment (MarcelMue) [inline integration/test/basic/basic_test.go]: Now using the version out of the `versionblob`.

File: .circleci/config.yml [hunk 3]
```diff
@@ -41,24 +41,52 @@ jobs:
         - ./chart-operator
         - ./architect
 
-  e2eTestBasicPR:
+  e2eTestCurBasicPR:
```
Comment (MarcelMue) [inline .circleci/config.yml]: Diff looks a bit strange, but I added the jobs aligned with `aws-operator` here.

File: integration/test/basic/basic_test.go (hunk 2 above)
Comment (rossf7) [inline integration/test/basic/basic_test.go]: <3

**Pull request merged**